eureka-client/src/main/java/com/netflix/appinfo/AmazonInfo.java

```diff
@@ -63,6 +63,8 @@
         availabilityZone("availability-zone", "placement/"),
         publicHostname("public-hostname"),
         publicIpv4("public-ipv4"),
+        spotInstanceTerminationTime("termination-time", "spot/"),
```

Comment:
very minor nit: mind calling this `spotTerminationTime`? That aligns more with how the rest of the enums are named.

Reply:
Sure. Changed to `spotTerminationTime`.